commands/summary.go

```diff
@@ -145,6 +143,14 @@ func addClientData(ctx context.Context, zipW *zip.Writer) {
 
 	addData(zipW, "client/pmm-admin-version.txt", now, bytes.NewReader([]byte(version.FullInfo())))
 
+	fileName := "pmm-admin-summary.zip"
+	err = downloadFile(fmt.Sprintf("http://%s:%d/logs.zip", agentlocal.Localhost, agentlocal.DefaultPMMAgentListenPort), fileName)
+	if err != nil {
+		logrus.Debugf("%s", err)
+		b = []byte(err.Error())
+	}
+	logrus.Info("pmm-admin-summary.zip created")
+
```

Comment:
we are on a right track, as a next step let's include files from logs.zip into main zip file

Reply:
you can find example of it in `addServerData`

Reply:
I did it